off-chain/test/Examples/Addition/ContractSeath.purs

```diff
@@ -24,58 +27,105 @@ import Seath.Test.Examples.Addition.Actions
   )
 import Seath.Test.Examples.Addition.Contract (initialContract)
 import Seath.Test.Examples.Addition.SeathSetup
-  ( Leader(Leader)
+  ( BlockhainState(..)
```

Comment:
For now, since Mlabs doesn't have a PureScript code style guide, the consensus is to use the same rules as in Haskell including the explicit import of the constructors. 

An advantage of this is that when you are working on something and the lsp  fails, you can still look for the right imported thing by searching for the word and looking in the imports, then use the local docs to get the info.